mlir/utils/performance/perfRunner.py

```diff
@@ -1214,6 +1224,9 @@ def main(args=None):
         configs = getConvConfigurations(paths.configuration_file_path)
     elif opType == Operation.GEMM:
         configs = getGemmConfigurations(paths.configuration_file_path)
+        # Make MLIR to be compatible with CK by forcing int8xint8->int8
+        if confClass == CKGemmConfig:
```

Comment:
I still think we are adding a further datatype in `perfRunner` (see my previous comment). Previously we had f32, f16, i8 and now we have f32, f16,i8i8, i8i32. Can we by default generate all the configuration, but give the user the possibility to pick which datatype they want to use?